swagger/api-manager.yaml

```diff
@@ -220,6 +220,9 @@ definitions:
       name:
         type: string
         pattern: '^[\w\d\-]+$'
+      kind:
```

Comment:
`kind` attribute should have `readOnly` attribute set to `true` (same for other swagger files)